tox.ini

```diff
@@ -12,12 +12,12 @@ commands = py.test --cov
 [testenv:lint]
 basepython = python3
 commands = flake8
-deps = -r requirements/lint.txt
```

Comment:
There is neither lint.in, nor lint.txt in the project. But check.in/check.txt files contain flake8.